app/jobs/school_data_importer_job.rb

```diff
@@ -1,5 +1,5 @@
 class SchoolDataImporterJob < CronJob
-  self.cron_expression = "0 0 * * *"
+  self.cron_expression = "0 0 4 * * * "
```

Comment:
Does this work? Cron doesn't normally do second resolution. The format I'm familiar with is `minute hour day_of_month month day_of_week`.

Reply:
Yeah, I tested it. The Cron implementation the gem uses is `second minute hour day_of_month month day_of_week` https://github.com/codez/delayed_cron_job/blob/master/lib/delayed_cron_job/cronline.rb

Reply:
That's helpful of them, given they link to a definition that doesn't do seconds in their docs...

Reply:
Ha! I didn't even see that. Might do an upstream PR to fix that.